force-app/main/default/applications/Events.app-meta.xml

```diff
@@ -12,6 +30,7 @@
     <label>Events</label>
     <navType>Standard</navType>
     <tabs>Public_Event__c</tabs>
+    <tabs>Event_Registration__c</tabs>
```

Comment:
@t-gul please remove event registration tab, users should not be able to create records in listview, they must create records using the button in related list of Event Registration  in Event Registration record detail page.